package com.example.demo.repo;

import com.example.demo.dto.BookDto;
import com.example.demo.entity.Book;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface BookRepository extends JpaRepository<Book,Long> {
    public Book getByBookId(String bookId);
}
